packages/firestore/test/integration/util/helpers.ts

```diff
@@ -59,8 +59,8 @@ import {
   TARGET_DB_ID,
   USE_EMULATOR
 } from './settings';
-import { _onRealtimePipelineSnapshot } from '../../../src/api/pipeline_impl';
 import { RealtimePipeline } from '../../../src/api/realtime_pipeline';
+import { onPipelineSnapshot } from '../../../src/api/reference_impl';
```

Comment:
@MarkDuckworth This import is suspicious, just to call it out.

Reply:
I'll take another look.